zhaquirks/tuya/tuya_trv.py

```diff
@@ -94,6 +95,7 @@ async def write_attributes(self, attributes, manufacturer=None):
     .applies_to("_TZE200_exfrnlow", "TS0601")
     .applies_to("_TZE200_9m4kmbfu", "TS0601")
     .applies_to("_TZE200_3yp57tby", "TS0601")
+    .replaces_endpoint(1, device_type=zha.DeviceType.THERMOSTAT)
```

Comment:
We can add a comment here about why we do this:
```suggestion
    # default device type is `SMART_PLUG` for this,
    # so change it back to keep UID/entity the same
    .replaces_endpoint(1, device_type=zha.DeviceType.THERMOSTAT)
```